### PR TITLE
en,zh: release note for v1.4.6

### PR DESCRIPTION
**Pull request by WangLe1321**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/releases/release-1.4.6.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+发布日期: 2023 年 7 月 19 日
+
+TiDB Operator 版本：1.4.6
+
+## 优化提升
+
+## 错误修复
+
+- 修复使用 >=v6.6.0 BR 做备份恢复时报 `Error loading shared library libresolv.so.2` 的错误 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```
Comment (csuzhangxc) [inline zh/releases/release-1.4.6.md:15]: - 修复使用 >=v6.6.0 BR 做备份恢复时报 `Error loading shared library libresolv.so.2` 的错误 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```suggestion
- 修复使用 >=v6.6.0 BR 做备份恢复时报 `Error loading shared library libresolv.so.2` 的错误 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
- 修复使用的 TiCDC image tag 不符合语义化版本时无法 Graceful Drain TiCDC 的问题 ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: zh/releases/release-1.4.6.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: TiDB Operator 1.4.6 Release Notes
+---
+
+# TiDB Operator 1.4.6 Release Notes
+
+发布日期: 2023 年 7 月 19 日
+
+TiDB Operator 版本：1.4.6
+
+## 优化提升
```
Comment (csuzhangxc) [inline zh/releases/release-1.4.6.md:11]:
```suggestion
## 优化提升

- 默认启用 volume resize 支持 ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```

File: en/releases/release-1.4.6.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+title: TiDB Operator 1.4.6 Release Notes
+---
+
+# TiDB Operator 1.4.6 Release Notes
+
+Release date: July 19, 2023
+
+TiDB Operator version: 1.4.6
+
+## Improvements
```
Comment (csuzhangxc) [inline en/releases/release-1.4.6.md:11]:
```suggestion
## Improvements

- Enable volume resize by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```

File: en/releases/release-1.4.6.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Bug fixes
+
+- Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```
Comment (csuzhangxc) [inline en/releases/release-1.4.6.md:15]:
```suggestion
- Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
- Fix the issue that graceful drain for TiCDC didn't work if a non-SemVer image tag used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.4.6.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+# TiDB Operator 1.4.6 Release Notes
+
+Release date: July 19, 2023
+
+TiDB Operator version: 1.4.6
+
+## Improvements
+
+- Enable volume resize by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```
Comment (ran-huang) [inline en/releases/release-1.4.6.md:13]:
```suggestion
- Enable volume resizing by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```

File: zh/releases/release-1.4.6.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+- 默认启用 volume resize 支持 ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
+
+## 错误修复
+
+- 修复使用 >=v6.6.0 BR 做备份恢复时报 `Error loading shared library libresolv.so.2` 的错误 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```
Comment (ran-huang) [inline zh/releases/release-1.4.6.md:17]:
```suggestion
- 修复使用 v6.6.0 及以上版本的 BR 执行备份恢复时报 `Error loading shared library libresolv.so.2` 的错误的问题 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```

File: en/releases/release-1.4.6.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+## Bug fixes
+
+- Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```
Comment (ran-huang) [inline en/releases/release-1.4.6.md:17]:
```suggestion
- Fix the issue of `Error loading shared library libresolv.so.2` when executing backup and restore with BR >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```

File: en/releases/release-1.4.6.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+- Fix the issue that graceful drain for TiCDC didn't work if a non-SemVer image tag used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (ran-huang) [inline en/releases/release-1.4.6.md:18]:
```suggestion
- Fix the issue that graceful drain for TiCDC does not work if a non-SemVer image tag is used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```

**Pull request merged**